blocks/card-social-comment/src/CardSocialComment.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import { ArrowForward, TurnRight, StarBorder, MoreHoriz } from "@mui/icons-material"
+import {
+  Card,
+  CardContent,
+  Box,
+  Avatar,
+  List,
+  ListItem,
+  Link,
+  ButtonGroup,
+  IconButton,
+  Typography,
+  ColorPaletteProp
+} from "@mui/joy";
+
+export default function CardSocialComment() {
+  const comments = [
+    {
+      commentor: 'Edward Clark',
+      commentAt: '14:07',
+      comments: [
+        <Typography>The biggest change of design in 2019 will be the makeup of the people doing the actual design work.</Typography>,
+        <Typography><Link underline="none" variant="soft" color="primary">@Steve</Link> please read this.</Typography>
+      ],
+      color: 'success',
+    },
+    {
+      commentor: 'Albert Ox',
+      commentAt: '13:37',
+      comments: [
+        <Typography>We absolutely need to get smarter about privacy. Society is creeped out by big tech and big data.</Typography>
+      ],
+      color: 'warning',
+    }
+  ]
```

Comment:
```suggestion
  ] as const
```

You can use `as const` so that you don't have to import `ColorPaletteProp` type.

Reply:
I've never used this.
This is so useful, Thank you so much.